### DISCUSSION TITLE
DigisparkKeyboard with MH-ET LIVE Tiny88

### DISCUSSION THREAD

**Discussion by kumpakan** [ArminJo/DigistumpArduino#46] [General]
Hi, I'm trying to use DigisparkKeyboard with MH-ET LIVE Tiny88. 
currently, I'm using Digistump AVR version 1.7.4 because version 1.7.5 doesn't support MH-ET LIVE Tiny88.
does Digistump AVR will support MH-ET LIVE Tiny88 again in the future, or how can I do in another way?
Thank you.

**Comment by ArminJo** [comment]
It is not planned to add MH-ET LIVE Tiny88 support in the future.
